R/tlf_ae_specific.R

```diff
@@ -75,16 +77,39 @@ tlf_ae_specific <- function(outdata,
   n_col <- ncol(tbl)
 
   # Define title
-  title <- collect_title(outdata$meta, outdata$population, outdata$observation, outdata$parameter, analysis = "ae_specific")
-  footnotes <- vapply(footnotes, glue::glue_data,
+  if(is.null(title)){
+    title <- collect_title(outdata$meta,
+      outdata$population,
+      outdata$observation,
+      outdata$parameter,
+      analysis = "ae_specific")
+  }
+
+ footnotes <- vapply(footnotes, glue::glue_data,
     .x = list(medra_version = medra_version), FUN.VALUE = character(1)
   )
   names(footnotes) <- NULL
 
   # Define column header
+
+  col_tbl <- strsplit(names(tbl), "_") |>
+    unlist() %>%
+    {.[. %in% c("n", "prop", "dur", "events")]} %>%
+    unique()
+
+  colhead_tmp <- paste(
+    sapply(X = col_tbl,
+      FUN = switch,
```

Comment:
would be good to have a way to map n, prop, dur, and events to user-provided labels, ideally ones that are provided in the meta definition, but alternatively in the final format_ae_XX call.